### ISSUE TITLE
Conversion error on rhel 6.7

### ISSUE THREAD

**Issue by msrubar** [OpenSCAP/openscap#251]
I faced an error while evaluating the pci-dss profile on rhel 6.7 using upstream's ssg and oscap.
```
cat /etc/redhat-release 
Red Hat Enterprise Linux Server release 6.7 (Santiago)
```

```
oscap xccdf eval --remediate --profile pci-dss --report /root/oscap_pci-dss_remediation_report.html scap-security-guide/RHEL/6/output/ssg-rhel6-xccdf.xml
```

```
....

itle^M Enable the NTP Daemon
Rule^M  service_ntpd_enabled
Ident^M CCE-27093-4
Ident^M DISA FSO RHEL-06-000247
/tmp/oscap.86u1ve/xccdf-result.xml:27283: parser error : PCDATA invalid Char value 27
      <message severity="info">Starting ntpd: ^[[60G[^[[0;32m  OK  ^[[0;39m]&#13;
                                              ^
/tmp/oscap.86u1ve/xccdf-result.xml:27283: parser error : PCDATA invalid Char value 27
      <message severity="info">Starting ntpd: ^[[60G[^[[0;32m  OK  ^[[0;39m]&#13;
                                                    ^
/tmp/oscap.86u1ve/xccdf-result.xml:27283: parser error : PCDATA invalid Char value 27
      <message severity="info">Starting ntpd: ^[[60G[^[[0;32m  OK  ^[[0;39m]&#13;
                                                                 ^
OpenSCAP Error: Conversion of the string "" to an integer (64 bits) failed: Invalid argument [oval_cmp.c:113]
Conversion of the string "" to an integer (64 bits) failed: Invalid argument [oval_cmp.c:113]
Conversion of the string "" to an integer (64 bits) failed: Invalid argument [oval_cmp.c:113]
Conversion of the string "" to an integer (64 bits) failed: Invalid argument [oval_cmp.c:113]
Conversion of the string "" to an integer (64 bits) failed: Invalid argument [oval_cmp.c:113]
Could not parse the XML document '/tmp/oscap.86u1ve/xccdf-result.xml' [oscapxml.c:429]
Result^M        fixed
```

Evaluation without remediation is fine.

**Comment by msrubar** [comment]
The issue was cause by PR [#870](https://github.com/OpenSCAP/scap-security-guide/pull/870) and fixed with PR [#909](https://github.com/OpenSCAP/scap-security-guide/pull/909).

**Comment by isimluk** [comment]
Thanks @msrubar.

The chances are that this problem will happen in reality often, so we should fix this in OpenSCAP I have filed #281 for to track it.
